Bundle/CoreBundle/Resources/script/linkModal.js

```diff
@@ -5,9 +5,12 @@ $(document).ready(function() {
         $('.modal').modal('hide').on('hidden.bs.modal', function(){$(this).remove()});
 
         var url = $(this).attr('href');
+        $('#canvasloader-container').fadeIn();
         $.get(url, function(data) {
             $('body').append(data);
+            $('#ajax-modal').modal('show');
```

Comment:
I'd rather to add boostrap modal function in the modal.html.twig file.
Take a look to  https://github.com/Victoire/victoire/blob/2.1/Bundle/CoreBundle/Resources/views/Layout/modal.html.twig#L18